docker/resources/common-scripts/kafka-propertiesSpec.json

```diff
@@ -0,0 +1,25 @@
+{
+  "prefixes": {
+    "KAFKA": false,
+    "CONFLUENT": true
+  },
+  "renamed": {
+    "KAFKA_ZOOKEEPER_CLIENT_CNXN_SOCKET": "zookeeper.clientCnxnSocket"
```

Comment:
I think we should skip zk since it's deprecated and will be removed in 4.0.

Reply:
Sounds good! We won't support Zookeeper in the docker image. I have updated the PR to reflect the same